modules/manage/pages/cluster-maintenance/compaction-tuning.adoc

```diff
@@ -0,0 +1,47 @@
+
+image::shared:compaction-example.png[Example of topic compaction]
+
+This diagram provides an illustration of a compacted topic. Imagine a remote sensor network that uses image recognition to track appearances of red pandas in a geographic area. The sensor network employs special devices that send messages to a topic when they detect one. You might enable compaction to reduce topic storage while still maintaining a record in the topic of the last time each device saw a red panda, perhaps to see if they stop frequenting a given area. The left side of the diagram shows all messages sent across the topic. The right side illustrates the results of compaction; older messages for certain keys are deleted from the message log.
+
+IMPORTANT:  When using Tiered Storage, compaction functions at the local storage level before segments are uploaded to storage. Once a segment is uploaded to storage it is not retrieved for further compaction operations. A key may therefore appear in multiple segments between Tiered Storage and local storage.
+
+While compaction reduces storage needs, Redpanda's compaction (just like Kafka's) does not guarantee perfect de-duplication of a topic. Duplicates of a message, or additional messages, may still exist within a topic. Compaction is not a complete topic operation; rather, it represents a best effort mechanism to reduce storage needs for topics created within a certain time window.
+
+== Configuring Cleanup Policy
+
+You typically configure compaction policy at the cluster level, but you can also specify an individual topic policy to override the cluster's setting. The cluster-level xref:reference:cluster-properties.adoc#_log_cleanup_policy[`log_cleanup_policy`] and the topic-level xref:reference:topic-properties.adoc#cleanuppolicy[`cleanup.policy`] support the following three options:
+
+* `delete`: Messages are deleted from the topic once the specified retention period (time and/or size allocations) is exceeded. This is the default mechanism and is analogous to disabling compaction.
+* `compact`: This triggers only cleanup of messages with multiple versions. A message that represents the only version for a given key is not deleted.
+* `compact,delete`: This combines both policies, deleting messages exceeding the retention period while compacting multiple versions of messages.
+
+== Compaction Policy Settings
+
+The various cleanup policy settings rely on proper tuning of a cluster's compaction and retention policy options. The applicable settings include the following:
+
+* xref:reference:cluster-properties.adoc#_log_compaction_interval_ms[`log_compaction_interval`]: Defines the compaction frequency in milliseconds. (default: 10,000ms)
```

Comment:
Same for the xrefs here, they don't seem to need the leading underscore for the section ID at least in the deploy preview, I'd double check

Reply:
I added in the anchors so I could remove underscores in my local build.